src/main/resources/sam.conf

```diff
@@ -290,5 +290,19 @@ resourceAccessPolicies {
         roles = ["support"]
       }
     }
+    spend-profile {
+      system {
+        memberEmails = [
+          ${?LEONARDO_PET_SERVICE_ACCOUNT}
+          ${?WSM_PET_SERVICE_ACCOUNT}
+        ]
+        descendantPermissions = [
+          {
+            resourceTypeName = "spend-profile",
+            roles = ["system"]
+          }
+        ]
+      }
+    }
```

Comment:
do you need the whole `descendantPermissions` stanza, or can this be just:
```
    spend-profile {
      system {
        memberEmails = [
          ${?LEONARDO_PET_SERVICE_ACCOUNT}
          ${?WSM_PET_SERVICE_ACCOUNT}
        ]
        roles = ["system"]
      }
    }
```

I honestly don't know, this is new to me

Reply:
`descendantPermissions` is required. That's what grants this role to all `spend-profile`s instead of granting the role only to the `resource_type_admin` resource

Reply:
the way to read is that this stanza is specifying a policy on a resource names `spend-profile` (line 293) of type `resource_type_admin` (line 262). The policy name is `system` (line 294). The rest is the same as the payload for the [overwrite policy api](https://sam.dsde-dev.broadinstitute.org/#/Resources/overwritePolicyV2).